-------------------------------------------------------------------------------
Test set: com.itau.dados_cadastrais.adapter.dynamo.exception.DynamoAdapterExceptionTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.030 s -- in com.itau.dados_cadastrais.adapter.dynamo.exception.DynamoAdapterExceptionTest
